code/modules/antagonists/changeling/changeling.dm

```diff
@@ -129,9 +129,11 @@
 
 /datum/antagonist/changeling/proc/regain_powers()//for when action buttons are lost and need to be regained, such as when the mind enters a new mob
 	emporium_action.Grant(owner.current)
-	for(var/path in purchasedpowers)
-		var/datum/action/changeling/S = new path()
-			S.Grant(owner.current)
+//	for(var/path in purchasedpowers)
```

Comment:
you know this is required code right?

Reply:
Yes it kills travis see the comments below it
It does nothing in testing also i wrote it so yes i know what it is